packages/runtime/container-runtime/src/gc/gcConfigs.ts

```diff
@@ -0,0 +1,170 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { assert } from "@fluidframework/common-utils";
+import { UsageError } from "@fluidframework/driver-utils";
+import { MonitoringContext } from "@fluidframework/telemetry-utils";
+import { IGCRuntimeOptions } from "..";
+import { IContainerRuntimeMetadata } from "../summary";
+import {
+	defaultInactiveTimeoutMs,
+	defaultSessionExpiryDurationMs,
+	disableTombstoneKey,
+	gcTestModeKey,
+	IGarbageCollectorConfigs,
+	oneDayMs,
+	runGCKey,
+	runSessionExpiryKey,
+	runSweepKey,
+	trackGCStateKey,
+} from "./gcDefinitions";
+
+/**
+ * Generates configurations for the Garbage Collector that it uses to determine what to run and how.
+ * @param gcOptions - The garbage collector runtime options.
+ * @param metadata - The container runtime's metadata.
+ * @param baseSnapshotGCVersion - The GC version of the base snapshot.
+ * @param existing - Whether the container is new or an existing one.
+ * @param mc - The monitoring context for reading configs from the config provider.
+ * @returns The garbage collector configurations.
+ */
+export function generateGCConfigs(
```

Comment:
Didn't we both say we like `Gc` better than `GC`? :P

Reply:
As long as it's consistent